File: app/src/main/java/com/foreverrafs/radiocore/activity/SplashActivity.java
package com.foreverrafs.radiocore.activity;

import android.content.Intent;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

public class SplashActivity extends AppCompatActivity {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        startActivity(new Intent(SplashActivity.this, MainActivity.class));
        finish();
    }
}

File: app/src/main/java/com/foreverrafs/radiocore/activity/SplashActivity.kt
package com.foreverrafs.radiocore.activity

import android.content.Intent
import android.os.Bundle

import androidx.appcompat.app.AppCompatActivity

class SplashActivity : AppCompatActivity() {
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        startActivity(Intent(this, MainActivity::class.java))
        finish()
    }
}

